app/src/main/java/edu/uc/zhukv/droneradarmap/Weather_Layer/TransparentTileOWM.kt

```diff
@@ -14,11 +14,14 @@ import java.net.URL
 class TransparentTileOWM(private val tileType: String):TileProvider {
     private val opacityPaint = Paint()
 
+    init {
+        setOpacity(50)
+    }
```

Comment:
Moved the init block up as recommended from: https://kotlinlang.org/docs/coding-conventions.html#class-layout